Src/Plugins/Tests.SalAnalyzer/SEC105.CorrectUseOfSalTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.SalAnalyzer
+{
+    public class CorrectUseOfSalTests : EndToEndTestsSalAnalyzer
```

Comment:
>CorrectUseOfSalTests [](start = 17, length = 20)

This plug in actually isn't about correcting use of SAL (there are definitely rule ideas for this!). Instead, it is about updating SAL v1 use to SAL v2. 

UpdateSalToCurrentVersion

or some such.

Reply:
Renamed.

---
In reply to: [604483673](https://github.com/microsoft/sarif-pattern-matcher/pull/347#discussion_r604483673) [](ancestors = 604483673)